packages/react-vega-demo/stories/ReactVegaLiteDemo.tsx

```diff
@@ -30,32 +31,47 @@ const data2 = {
   ],
 };
 
-const spec1: VisualizationSpec = {
+const convertToVegaSpec = (spec: TopLevelSpec) => {
+  const vegaSpec = compile(spec).spec;
+
+  vegaSpec.signals = (vegaSpec.signals || []).concat({
```

Comment:
Yes, this is a workaround. Because in the demo part there's a 'hover' signal listener, yet Vega-Lite doesn't support signal at all, so I want to show others how to use signal with Vega-Lite specs.

Reply:
Vega-Lite compiles to Vega and does use signals. 

This change feels out of place for a react specific demo.